deploy/examples/jbpm-quarkus-example-persistence.yaml

```diff
@@ -5,9 +5,6 @@ metadata:
 spec:
   build:
     env:
-      # this value is just an example, set yours (optional)
-      - name: MAVEN_MIRROR_URL
-        value: "http://nexus3.nexus.svc.cluster.local:8081/repository/maven-public/"
```

Comment:
Same here.

Reply:
there is a example already below

Reply:
I was just thinking about environment variables in general, but probably can be added ad-hoc.